src/UALJs.ts

```diff
@@ -99,7 +98,7 @@ export class UALJs extends UAL {
     const sessionExpiration = localStorage.getItem(UALJs.SESSION_EXPIRATION_KEY) || null
     if (sessionExpiration) {
       // clear session if it has expired and continue
-      if (Number(sessionExpiration) < new Date().getTime()) {
+      if (new Date(sessionExpiration) <= new Date()) {
```

Comment:
Adding the `=` technically changes the logic here, so I'd expect some sort of test to be added specifically for that. Instead of having to test this function multiple times (looks like there's a ton of dependencies), this logic could be split out in to its own function and tested independently.

Reply:
I added a test that confirms that a zero second `shouldInvalidateAfter` value will invalidate a session from re-authenticating.  I could also remove my `=` because it could be open to debate whether it expires on or after the exact second.

Reply:
New test: https://github.com/EOSIO/ual-plainjs-renderer/pull/71/commits/e36dc13c5477dc8bb45906743dc0504aada65670